packages/squiggle-lang/src/rescript/Distributions/DistributionTypes.res

```diff
@@ -92,7 +92,9 @@ module DistributionOperation = {
     | ToString
     | ToSparkline(int)
 
-  type toScore = KLDivergence(genericDist) | LogScore(float, option<genericDist>)
+  type scoreDistOrScalar = Score_Dist(genericDist) | Score_Scalar(float)
```

Comment:
I imagine this should be called ``Dist`` and ``Scalar`` (or Float, which is more common). Ideally they would be in a module, and not used by any other files, so a generic name wouldn't be problematic.